packages/lit-ssr/src/test/integration/tests/basic.ts

```diff
@@ -0,0 +1,3829 @@
+/**
```

Comment:
This module is way too big. Please break into separate test modules for the different types.

Reply:
The integration test runner is bespoke for SSR, so it will take some infrastructure work to re-factor it to handle multiple files; before we go further down a bespoke setup, we should consider whether we can pivot to wtr configured with the proxies, etc. needed to run the SSR tests.

This PR is mostly trying to port the `lit-ssr` work over without large changes, so I suggest breaking test infrastructure out to a separate issue, filed here: https://github.com/Polymer/lit-html/issues/1378